purchases/src/main/kotlin/com/revenuecat/purchases/CoroutinesExtensionsCommon.kt

```diff
@@ -30,6 +30,27 @@ suspend fun Purchases.awaitOfferings(): Offerings {
     }
 }
 
+/**
+ * Fetch the configured offerings for this users. Offerings allows you to configure your in-app
+ * products via RevenueCat and greatly simplifies management. See
+ * [the guide](https://docs.revenuecat.com/offerings) for more info.
+ *
+ * Offerings will be fetched and cached on instantiation so that, by the time they are needed,
+ * your prices are loaded for your purchase flow. Time is money.
+ *
+ * Coroutine friendly version of [Purchases.getOfferings].
+ *
+ * @return The [Result] of [Offerings] available to this user.
+ */
+@JvmSynthetic
+suspend fun Purchases.awaitOfferingsResult(): Result<Offerings> =
```

Comment:
Can be done separately, but should we do the same thing for the `defaults` flavor methods: https://github.com/RevenueCat/purchases-android/blob/main/purchases/src/defaults/kotlin/com/revenuecat/purchases/coroutinesExtensions.kt

Reply:
Oh, I missed that! Let me work on this separately after merging this. Thank you for the details! 🙌